thread-discussions.mdx

```diff
@@ -13,11 +13,20 @@ For your convenience, the discussion will also be visible in the Plain thread.
 
 To enable this functionality, you will need to setup your [Slack integration](/slack) and add the Plain bot to your channel.
 
+<Warning>
+  When Plain is added to a channel, we automatically classify it as a [Customer channel](/slack/how-slack-messages-create-threads) if it's a slack connect channel or one for internal discussions otherwise.
```

Comment:
```suggestion
  When Plain is added to a channel, we automatically classify it as a [Customer channel](/slack/how-slack-messages-create-threads) if it's a _Slack Connect_ channel or one for internal discussions otherwise.
```